src/Sarif.Multitool/ResultMatchSetCommand.cs

```diff
@@ -64,7 +65,11 @@ public int Run(ResultMatchSetOptions options)
                         if (mergedLog.Runs[0].Results.Any(r => r.BaselineState != BaselineState.Unchanged))
                         {
                             string outputFilePath = Path.Combine(options.OutputFolderPath, fileName);
-                            WriteSarifFile(_fileSystem, mergedLog, outputFilePath, formatting);
+
+                            if (DriverUtilities.ReportWhetherOutputFileCanBeCreated(outputFilePath, options.Force, _fileSystem))
```

Comment:
>ReportWhetherOutputFileCanBeCreated [](start = 48, length = 35)

Is it OK that the function 'succeeds' even when it can't write one or more files?

Reply:
Not really, no :-) Fixed.

---
In reply to: [312283939](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r312283939) [](ancestors = 312283939)